management_interface/internal_integrations/management_api/client.py

```diff
@@ -0,0 +1,58 @@
+import uuid
+from datetime import date
+from typing import Optional
+
+import requests
+from fhir.resources.operationoutcome import OperationOutcome
+
+from internal_integrations.management_api.exceptions import ManagementAPIClientError
+
+from internal_integrations.management_api.settings import get_management_api_settings
+
+
+class ManagementAPIClient:
+    def __init__(self, base_url: Optional[str] = None, session: Optional[requests.Session] = None):
+        self.base_url = base_url or get_management_api_settings().base_url
+        self.session = session or requests.Session()
+
+    def create_subscription(self, patient_given_name: str, patient_family_name: str, nhs_number: str,
+                            birth_date: date) -> uuid.UUID:
+        body = {
+            "resourceType": "Patient",
+            "identifier": [
+                {
+                    "system": "https://fhir.nhs.uk/Id/nhs-number",
+                    "value": nhs_number,
+                    "extension": [
+                        {
+                            "url": "https://fhir.hl7.org.uk/StructureDefinition/Extension-UKCore-NHSNumberVerificationStatus",
+                            "valueCodeableConcept": {
+                                "coding": [
+                                    {
+                                        "system": "https://fhir.hl7.org.uk/CodeSystem/UKCore-NHSNumberVerificationStatusEngland",
+                                        "code": "03",
+                                        "display": "Trace required"
+                                    }
+                                ]
+                            }
+                        }
+                    ]
+                }
+            ],
+            "name": [
+                {
+                    "use": "official",
+                    "family": patient_family_name,
+                    "given": [
+                        patient_given_name,
```

Comment:
At this point it should be a list of given names, not just a single given name (e.g. Henry David should become "Henry", "David")

Reply:
Hmm, how do you want this to be displayed as an input? One text field, then splitting by space?

Reply:
@michal-kras can we have one text field and split by spaces - this will match the bulk upload behaviour

Reply:
@regularfry CSV upload hasn't been merged when I was developing this, I will rebase and see what can be done.

Reply:
Yep, no worries - perils of not integrating often enough.  It happens.